judge/admin/contest.py

```diff
@@ -130,6 +129,18 @@ class ContestAdmin(VersionAdmin):
     filter_horizontal = ['rate_exclude']
     date_hierarchy = 'start_time'
 
+    def get_actions(self, request):
```

Comment:
Actually, this is the wrong condition. They should still be allowed to make organization- or user-private contests visible.